src/modules/shared/components/MediaCard/MediaCard.types.ts

```diff
@@ -17,6 +17,7 @@ export interface MediaCardProps {
 	title?: string | ReactNode;
 	type: MediaTypes;
 	view?: MediaCardViewMode;
+	hasChildren?: boolean;
```

Comment:
maybe rename this to hasRelated
since we don't know if they are children until meemoo adds the parent child indicator. And hasChildren might be confusing when we implement the parent-child feature in the future.